### PR TITLE
add edit icon to disable notes section

### PR DESCRIPTION
**Pull request by kwiedenhaupt**
Screen Shot 2022-10-29 at 9 07 37 PM

1. added ArrowRightCircle icon to notes section 
2. added local state editGoalNotes to make icon yellow when field is editable and grey when disabled.

### CODE REVIEW

File: pages/goals/[slug].tsx [hunk 1]
```diff
@@ -80,11 +82,36 @@ const EditGoalsPage = () => {
...
               }}
             />
+
             {editedGoalValues.goalName.length > 60 && (
               <p className="text-xs text-red-600">
                 please keep your goal under 60 characters
               </p>
             )}
+            <div className="flex">
+              <ArrowCircleRightIcon
```
Comment (iamnotreddy) [inline pages/goals/[slug].tsx:92]: - add a hover state
- move icon to the right of the section title
- use the same icon from the main goals page

**Review by iamnotreddy — COMMENTED** [review]
also let's add `isArchived: {_eq: false}, isComplete: {_eq: false}` to the where clause of the FETCH_USER_GOALS_COUNT query

File: pages/goals/[slug].tsx (hunk 1 above)
Comment (kwiedenhaupt) [reply pages/goals/[slug].tsx]: updated PR to fix these issues!

**Review by iamnotreddy — APPROVED** [review]
I added a small commit to only highlight goals if they are in the current section; also adjusted the threshold parameters a bit and changed the styling to highlight goals yellow if 7 days are remaining, red if it's overdue. LMK if you think this highlighting is too bold / hard to read 

Looks like you need to fix a merge conflict, otherwise looks great! 

Screen Shot 2022-10-31 at 8 20 50 PM

**Comment by kwiedenhaupt** [comment]
I really love the changes you made!
my personal aesthetic would be bg-red-300 and bg-yellow-200 because it is easier to read lol but let me know if you'd like me to change or just leave it and then I'll squash and merge! fixed the merge commits!
Screen Shot 2022-11-01 at 4 07 15 PM

**Pull request merged**